physical_infra_provider_configuration/automation/topics/creating_configuring_assigning_policy.adoc

```diff
@@ -0,0 +1 @@
+=== Creating, Configuring and Assigning a new Policy
```

Comment:
@skovic. I've created this empty sub section for better organize, like this example:
![image](https://user-images.githubusercontent.com/12627705/31893552-36de008a-b7e2-11e7-8604-0be1e6d41f0c.png)

Reply:
OK, got it, thanks